dialogue-target/src/main/java/com/palantir/dialogue/Request.java

```diff
@@ -186,7 +182,7 @@ private Builder() {}
         }
 
         public Request.Builder putQueryParams(String key, String... values) {
-            mutableQueryParams().putAll(key, values);
+            mutableQueryParams().putAll(key, Arrays.asList(values));
```

Comment:
should we avoid the extra allocation and iterate over the array calling put? elsewhere you seem to suggest that is more performant than putAll

Reply:
I don’t think this method is ever used

Reply:
ok, should we mark it as deprecated then?

Reply:
Eh, I wouldn't go though the trouble at this point, but I don't have a strong preference